eng/common/pipelines/templates/steps/write-filesystemmetrics.yml

```diff
@@ -0,0 +1,6 @@
+steps:
+    - task: Powershell@2
+      inputs:
+        filePath: $(Build.SourcesDirectory)/eng/common/scripts/Write-FileSystemMetrics.ps1
+        pwsh: true
+      displayName: Write filesystem metrics
```

Comment:
The step is to log disk space, so it is better to fail silently. 
Better to add `continueOnError` here.

Reply:
Added, thanks